tests/modules/programs/gnome-terminal/gnome-terminal-1.nix

```
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    nixpkgs.overlays = [
+      (self: super: {
+        gnome3.gnome-terminal =
+          pkgs.writeScriptBin "dummy-gnome3-gnome-terminal" "";
```

```suggestion
        gnome.gnome-terminal =
          pkgs.writeScriptBin "dummy-gnome3-gnome-terminal" "";
```

Sorry, last minute request.
I was looking at the logs to verify whether additional dependencies were being fetched during the test run.
`gnome3` has been renamed to `gnome`.

---

done